### PR TITLE
Update README.md for v3

### PR DESCRIPTION
**Pull request by lthibault**
I've taken the liberty of cleaning up the README a bit in preparation for `v3`.  In particular, certain details of the installation process are no longer relevant with the upcoming switch to modules (see: #165, #166).

Additionally, I've made some rather opinionated changes with respect to "marketing" (or -- if you prefer -- with respect to the ordering of information by priority).  I'm of the opinion that README's should first and foremost answer the question "what is this software, and is it for me?".  The answers to these questions are somewhat scattered  and buried below technical details, which IMHO only become relevant _after_ the initial what/why questions have been answered.

My basic approach can be summarized as follows:

1. In the heading, briefly what Capnp is, what go-capnp is, and why you might want to use it.
2. If you're still reading, your next question is likely to be "how do I get it"? ==> installation
3. If you successfully installed, your next question is "how do I start using it?"  ==> getting started
4. Once you have a minimal working example, you probably want to do something specific ==> API ref (godoc)
5. Once you've mapped the docs onto a proof-of-concept in your head, you're wondering if there are caveats/gotchas ==> API compatibility notice + license

Feedback very much welcome.

P.S.:  I added a couple of badges because nothing says "serious project" like badges.  Devs love badges. 😝

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,55 @@
 # Cap'n Proto bindings for Go
 
-[![GoDoc](https://godoc.org/zombiezen.com/go/capnproto2?status.svg)][godoc]
-[![Build Status](https://travis-ci.org/capnproto/go-capnproto2.svg?branch=master)][travis]
+[![GoDoc](https://godoc.org/capnproto.org/go/capnp/v3?status.svg)][godoc]
+![License](https://img.shields.io/badge/license-MIT-brightgreen?style=flat-square)
```
Comment (zenhack) [inline README.md:4]: Tangentially related, it appears GitHub's auto-detection of licenses can't work out which license we use, probably due to formatting oddities.

File: README.md [hunk 2]
```diff
@@ -1,66 +1,55 @@
...
-- a Go package that provides runtime support
-- a Go package that implements Level 1 of the RPC protocol
+[Cap’n Proto](https://capnproto.org/) is an insanely fast data interchange format similar to [Protocol Buffers](https://github.com/protocolbuffers/protobuf), but much faster.
 
-[godoc]: https://godoc.org/zombiezen.com/go/capnproto2
-[travis]: https://travis-ci.org/capnproto/go-capnproto2
+It also includes a sophisticated RPC system based on [Object Capabilities](https://en.wikipedia.org/wiki/Object-capability_model), ideal for secure, low-latency applications.
 
-## Getting started
+### This package provides:
```
Comment (zenhack) [inline README.md:11]: Nitpick: jumping from an `<h1>` to an `<h3>` is a bit ugly. I don't think this necessarily needs to be a heading, even.

**Review by zenhack — APPROVED** [review]
Looks reasonable to me; a couple comments in-line but nothing I'd block over.

File: README.md (hunk 1 above)
Comment (lthibault) [reply README.md:4]: What would be the course of action, here?

Comment (zenhack) [reply README.md:4]: We could futz with it, but I don't care that much, just something I noticed when I saw this.

**Pull request merged**